src/Workspaces/Core/Portable/Workspace/DocumentActiveContextChangedEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    public sealed class DocumentActiveContextChangedEventArgs : EventArgs
+    {
+        public Workspace Workspace { get; }
+        public SourceTextContainer SourceTextContainer { get; }
+        public DocumentId PreviousActiveContextDocumentId { get; }
+        public DocumentId CurrentActiveContextDocumentId { get; }
```

Comment:
After the move to "previous / current" I looked through the code base to make sure we were consistent on parameter ordering.  In "old / new" parameters are always expressed in that ordeer: old then new.  Wanted to make sure that we did the same for "previous / current". 

Looking though it doesn't seem that we use "previous / current" pairings in our code base.  It's possible I missed it but I went through every usage of previous I could find in source.roslyn.io.  Hence it seems this is a brand new pattern.  Are we still comfortable with it?  

@mattwar, @dpoeschl